server.js

```diff
@@ -0,0 +1,102 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  const uri = request.url;
+  const reqMethod = request.method;
+  const reqVersion = request.httpVersion;
+
+  function readF(fileLocation, conType, statusCode, reasonPhrase) {
+    fs.readFile(fileLocation, (err, data) => {
+      if (err) throw err;
+      response.setHeader(`Content-Type`, conType);
+      response.setHeader(`Content-Length`, data.length);
+      response.writeHead(statusCode, reasonPhrase);
+      response.write(data);
+      response.end();
+    });
+  };
+
+  // function writeF(fileLocation, conType, statusCode, reasonPhrase) {
+  //   fs.writeFile(fileLocation, (err, data) => {
+  //     if (err) throw err;
+  //     response.setHeader(`Content-Type`, conType);
+  //     response.setHeader(`Content-Length`, data.length);
+  //     response.writeHead(statusCode, reasonPhrase);
+  //     response.write(data);
+  //     response.end();
+  //   });
+  // };
+
+  //testing nested switch cases
+  switch (reqMethod) {
+    case 'GET':
+      switch (uri) {
```

Comment:
You do not want to use a `switch case` for your `uri` values. The list of available resources will change over the life of the program, and you will not be able to add new resources. Pass the uri in to your `getHandler` function and add the logic in there to retrieve whatever is requested.